### PR TITLE
feat(game-refs/vehicle-models): now using html to draw vehicle models

### PR DESCRIPTION
**Pull request by ahcenezdh**
Instead of using a markdown table, we are now using HTML/CSS to display every vehicle model

image

All vehicles are sorted by their categories
image

I sent a message to @fengorian and am waiting for his feedback

All vehicles data were generated with a nodeJS script using this <a href="https://github.com/DurtyFree/gta-v-data-dumps/blob/master/vehicles.json">list</a>

### CODE REVIEW

**Comment by ahcenezdh** [comment]
If you want to build the docs, see #417

**Comment by AvarianKnight** [comment]
Why not keep the `height` property as I assume it will fix the 

> Some vehicle cards are smaller or bigger than others. If someone has a fix, I'm open to suggestions

**Comment by ahcenezdh** [comment]
> Why not keep the `height` property on the img, as I assume it will fix the
> 
> > Some vehicle cards are smaller or bigger than others. If someone has a fix, I'm open to suggestions

Should be good

**Comment by PsychoShock** [comment]
While reworking the page, adding vehicles supported in each builds could be a nice feature. That way, announcements could refer to this doc page on release.

**Comment by ahcenezdh** [comment]
> While reworking the page, adding vehicles supported in each builds could be a nice feature. That way, announcements could refer to this doc page on release.

I don't know for this, @4mmonium is it ok for you?

File: content/docs/game-references/vehicle-models.md [hunk 1]
```diff
@@ -3,852 +3,6997 @@ title: Vehicle models
...
+    .category h2 {
+        text-align: left; 
+        margin-top: 30px; 
+        margin-bottom: 10px; 
+        padding-left: 20px; 
+    }
+
+    .vehicles {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, minmax(180px, 1fr));
```
Comment (AvarianKnight) [inline content/docs/game-references/vehicle-models.md:29]:
```suggestion
        grid-template-columns: repeat(4, 180px);
```
Workaround for hover css in the docs causing the auto-fill to only show one car per line on hover

File: content/docs/game-references/vehicle-models.md [hunk 2]
```diff
@@ -3,852 +3,6997 @@ title: Vehicle models
...
+        gap: 20px;
+        padding: 20px;
+        justify-content: center;
```
Comment (AvarianKnight) [inline content/docs/game-references/vehicle-models.md:32]:
```suggestion

```
Workaround for the front-end trying to fit content on hover of the first item causing the layout to shift to the left constantly.

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

**Comment by AvarianKnight** [comment]
Pushed a change to use markdown headers instead of h2's so we don't need the extra table of contents at the top

image

**Comment by AvarianKnight** [comment]
There's currently a few vehicles missing images

- astron2
- arbitergt
- cyclone2
- ignus2
- everon

**Comment by ahcenezdh** [comment]
> There's currently a few vehicles missing images
> 
> * astron2
> * arbitergt
> * cyclone2
> * ignus2
> * everon

For vehicles images, you have to see with @fengorian since those images were made by him

**Pull request merged**